zebra-state/src/service/finalized_state.rs

```diff
@@ -343,6 +353,12 @@ impl FinalizedState {
         let _ = stdout().lock().flush();
         let _ = stderr().lock().flush();
 
+        // Give some time to logger thread to flush out any remaining lines to stdout
```

Comment:
```suggestion
        // Give some time to logger thread to flush out any remaining lines to stdout
        // and yield so that tests pass on MacOS
```